### DISCUSSION TITLE
Is there a good way to just examine a token without caring about the signature?

### DISCUSSION THREAD

**Discussion by bclouser** [golang-jwt/jwt#122] [Q&A] [answered]
It looks like this used to exist with the ParseUnverified function, but that doesn't exist anymore...

I just need to yank out a claim from a token i know is valid and so I really don't care about the whole signature-validation dance... ya feel?

Kind regards,
Ben

**Comment by oxisto** [comment, marked as answer]
The `ParseUnverified` function still exists https://pkg.go.dev/github.com/golang-jwt/jwt/v4#Parser.ParseUnverified. Not sure what makes you think otherwise.

Be aware though you should REALLY really really only use this if you know what you are doing, i.e. verified this token beforehand.

**Reply by bclouser** [reply]
yeahhhhhh, i saw this the next day with fresh eyes. Sorry about the question.
